src/canisters/frontend/ic-certified-assets/assets.did

```diff
@@ -77,6 +87,12 @@ type StreamingStrategy = variant {
   };
 };
 
+type AssetProperties = record {
+  max_age: opt nat64;
+  headers: opt vec HeaderField;
+  redirect: opt AssetRedirect
```

Comment:
```suggestion
  redirect: opt vec AssetRedirect
```
What do you think about this? I think it would make sense to be able to define multiple redirect rules, tho that may affect performance when someone adds too many rules. But then, the only case were I think it would be useful is testing in local env, e.g.
```json
[
  {
    "match": "**/*", 
    "redirects": [
       {"from": {"host": "localhost:3333"}, "to":{"host": "somewhereelse.com"}},
       {"from": {"host": "raw.ic0.app"}, "to":{"host": "ic0.app"}},
    ]
  }
]
```

Reply:
or the following use case:
- redirect all user-generated traffic from raw to non-raw
- redirect all bot-generated traffic from non-raw to raw

i think this would require two separate redirect rules